pom.xml

```
@@ -12,7 +12,7 @@
     <version>3.2.0-SNAPSHOT</version>
     <packaging>hpi</packaging>
     <properties>
-        <bitbucket.version>5.5.9</bitbucket.version>
+        <bitbucket.version>6.10.17</bitbucket.version>
```

```suggestion
        <bitbucket.version>6.10.0</bitbucket.version>
```
As far as I know there are no known issues (that affects our plugin) in `6.10.0`? If we put this at `6.10.17` the readme must reflect that. I'm happy either way, I think 6.10.0 is the easiest thing for us here.